dsl/scripts/jobs.groovy

```diff
@@ -436,10 +436,10 @@ def jobs = [
             ANSIBLE_PLAYBOOK_COMMON: 'playbook-barenode-common.yaml',
         ],
         parameters: vegavisorRestartNetworkParams(
-            'TOP_UP_BOTS': true,
-            'USE_CHECKPOINT': false,
-            'CREATE_MARKETS': true,
-            'NODE_LABEL': 's-4vcpu-8gb',
+            TOP_UP_BOTS: true,
+            USE_CHECKPOINT: false,
+            CREATE_MARKETS: true,
+            NODE_LABEL: 's-4vcpu-8gb',
```

Comment:
I think for restart network we can also reduce size, but it is fine to use this instance here. The release job is mainly starts the ansible